docs/user-guide/styling-your-docs.md

```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```

Comment:
@d0ugal I've only just started using MkDocs and have found most of the documentation to be good quality. Ping me if you want a proofreader any time!